activation/activation.go

```diff
@@ -1019,23 +981,17 @@ func (b *Builder) getPositioningAtx(
 
 func (b *Builder) Regossip(ctx context.Context, nodeID types.NodeID) error {
 	epoch := b.layerClock.CurrentLayer().GetEpoch()
-	atx, err := atxs.GetIDByEpochAndNodeID(b.db, epoch, nodeID)
+	id, blob, err := atxs.AtxBlob(b.localDB, epoch, nodeID)
```

Comment:
I'm not sure I agree with this change. That would mean all ATXs published via the node are stored twice - once in the state and once in the local DB. Also the local DB can (theoretically) be deleted by the operator if they want to reset the node without having to resync. With this change would that mean the ATX might be deleted before it was ever gossiped?

Reply:
This is to support the "node split". When it runs as the "activation-service", it doesn't have the state DB (it's kept on the "node service"). So the idea was to keep own (maybe only the last one) ATX in the local DB for regossiping purposes. Perhaps you have a better idea?

Reply:
Hm.. you are right that the activation service needs to persist local state if run independently of the node 🤔  I guess it's OK then.

Maybe we should make clear that `atxs.AtxBlob` is different from other `atxs.*` functions to make that clearer? How about having a `localatxs` package for that purpose?